### ISSUE TITLE
A7 - rename to "Insufficient Detection and Response"

### ISSUE THREAD

**Issue by vanderaj** [OWASP/Top10#41]
Emailing you directly just because i'm too lazy to sign up for a mailing list I know I probably won't read. These aren't private comments, I'm happy for them to be made public or forwarded to the mailing list or whatever.
The content of the Top 10 2017 RC look great. I completely agree with the changes, with the one exception...
 ... the title of A7 "Insufficient Attack Protection", doesn't really match the content:
Detection, Response and Remediation aren't protective or preventative measures, they're responsive measures
"Insufficient Attack Protection" on naive reading (without reading the risk content), sounds like it could mean "you haven't bought enough vendor shit to protect your website". This is misleading, and could be misused
In it's place, I'd like to see something like "Insufficient Attack Response" or something similar.
Actually this is a way better suggestion than mine https://twitter.com/mik235/status/852055872234770432

9.	Liam Oshan

**Comment by davewichers** [comment]
I think Insufficient Attack Detection & Response would be a better title that's more obvious to the reader. It's longer, which is why it wasn't used originally, but it's probably worth it for the clarity.

**Comment by Neil-Smithline** [comment]
From @danielmiessler

I’m going to make some comments about the proposed 2017 update of the flagship OWASP Project—the OWASP Top 10.

Before I do, I just want to say that as a present and former leader of multiple OWASP projects (IoT Security, Mobile Top 10, Game Security Framework, etc.) over the last seven years, I empathize with the difficulty of making these lists. It sucks to work for weeks, months, and sometimes years to get consensus on something, only to have 1,000 internet randos fly by and crap on it. Please accept my comments as constructive criticism rather than hate. I appreciate what you do.

INITIAL IMPRESSIONS AND ANALYSIS

So here are some of my thoughts on the new proposed organization of the categories.

I HATED A7 INITIALLY

My averse reaction to it was immediate, negative, and strong. It’s called Insufficient Attack Protection, which struck me as just a horrible name. The reason for this I’ll explain more in my next point. But then I read the full page description on it, and I definitely see where they were going with it.

It’s basically detection and response (which includes patching), which I think is a phenomenal addition to the list. I do dislike the name, though. So I started thinking about better names, as I’ve done a million times in my various OWASP projects, and quickly realized all the better names are four to five words long.

Insufficient Detection and Response
Lack of Detection, Response, and Patching
Etc.
So they probably battled over this point for days or weeks or months, only to arrive at a not great solution of, “Insufficient Attack Protection”. It’s a noble attempt, although I might still try to convince them still to change it to, Insufficient Detection / Response.

Anyway, short version is I love the category but really dislike the name.

**Comment by vanderaj** [comment]
This risk is going to be removed as per community feedback at the Project Summit, survey data and revised data call. Thank you for your input on this matter.
